.github/workflows/run-td-build.yml

```diff
@@ -6,10 +6,6 @@ on:
   push:
     branches:
       - main

```

Comment:
there a reason for this yml edit? should we not be fetching strings from these other branches?

Reply:
I think all string changes should be done on main and be cherry-picked into the release branch. It's simpler.
String changes on release branches and the corresponding string translations don't ever make it back into main.

TD looks at our English strings on every push and has translations for each of them in their cache. If we have differing English strings in main vs a release branch, the translations in the cache will be on the latest push. I would like to have the translations always be for the English strings on main.